lib/ssm_config.rb

```diff
@@ -1,12 +1,14 @@
 class SsmConfig
   VERSION = '0.1.1'.freeze
   CONFIG_PATH = 'config'.freeze
-
+  TABLE_NAME = 'ssm_config_records'.freeze
   class << self
     def method_missing(meth, *args, &block)
       config_file = Rails.root.join(CONFIG_PATH, "#{meth}.yml")
 
-      if File.exist?(config_file)
+      if ActiveRecord::Base.connection.table_exists? TABLE_NAME
+        return {}
```

Comment:
we should also be at least checking that the key for the requested file exists. Right now if we create just the table itself we will always be getting an empty hash. This might be OK, but I'd rather make sure we never break any functionality when merging code.